tests/components/unifi/test_config_flow.py

```diff
@@ -236,12 +240,12 @@
 
 async def test_option_flow(hass):
     """Test config flow options."""
-    entry = MockConfigEntry(domain=config_flow.DOMAIN, data={}, options=None)
-    hass.config_entries._entries.append(entry)
+    controller = await setup_unifi_integration(hass, wlans_response=WLANS)
 
     flow = await hass.config_entries.options.async_create_flow(
```

Comment:
Try to use the core interfaces instead, `hass.config_entries.options.async_init`.

Reply:
Fixed it, thanks Martin!